### PR TITLE
Localise Gujarati GU initial commit

### PR DESCRIPTION
**Pull request by harsh4870**
Signed-off-by: Harsh Manvar [harsh.manvar111@gmail.com](mailto:harsh.manvar111@gmail.com)
Localizes README-gu.md to Gujarati
Localizes config.toml to Gujarati
Localizes content/gu/OWNERS to Gujarati
Localizes content/gu/_common-resources/index.md to Gujarati
Localizes content/gu/_index.html to Gujarati
Localizes content/gu/case-studies/_index.html to Gujarati
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/kubernetes/website/issues/38740

### CODE REVIEW

File: content/gu/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+તમારે કુબરનેટ્સ ક્લસ્ટર અને ક્યુબે સીટીએલ કમાન્ડ લાઇન ટૂલની જરૂર પડશે
+રૂપરેખાંકિત હોવું આવશ્યક છે જેથી તેઓ તમારા ક્લસ્ટર સાથે વાતચીત કરી શકે। અમે ભલામણ કરીએ છીએ કે તમે આ ટ્યુટોરીયલને ક્લસ્ટરમાં ચલાવવા માટે ઓછામાં ઓછા બે નોડ્સનો ઉપયોગ કરો જે કંટ્રોલ પ્લેન જેમ કાર્ય ના કરે.। જો તમારી પાસે પહેલેથી જ ક્લસ્ટર નથી, તો તમે 
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
+ની મદદથી તે બનાવી શકે છે, અથવા તમે નીચેના બે કુબરનેટ્સ પ્લેગ્રાઉન્ડ્સનો ઉપયોગ કરી શકો છો :
+
+* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
```
Comment (utkarsh-singh1) [inline content/gu/includes/task-tutorial-prereqs.md:6]: Katacoda playground support for kubernetes has ended.

**Comment by utkarsh-singh1** [comment]
Instead of Katacoda you can use [Killercoda Playgrounds](https://killercoda.com/playgrounds/scenario/kubernetes).

**Comment by sftim** [comment]
@harsh4870 how is this work going? Did you form a localization team for Gujarati?

**Comment by harsh4870** [comment]
Hi @sftim yes we are working on it, little slow but sure progress

**Comment by sftim** [comment]
Any update @harsh4870? It'd be great if the team would at least check in on this PR so we know the work is ongoing.

**Comment by T-Lakshmi** [comment]
@harsh4870,
Are you still working on this PR? Please update.

**Comment by harsh4870** [comment]
Hi @T-Lakshmi  yes but taking longer than expected time, as i am only one contributing so

**Pull request closed without merging**